libraries/app/database_api.cpp

```diff
@@ -1018,7 +1018,7 @@ market_ticker database_api_impl::get_ticker( const string& base, const string& q
    result.lowest_ask = 0;
    result.highest_bid = 0;
 
-   auto price_to_real = [&]( const share_type a, int p ) { return double( a.value ) / pow( 10, p ); };
+//   auto price_to_real = [&]( const share_type a, int p ) { return double( a.value ) / pow( 10, p ); };
```

Comment:
That is correct. I noticed that it was an unused variable. Do you prefer that I remove it from the commit?